tasks/reference.js

```diff
@@ -76,7 +76,7 @@ function determineCategory(tag){
             return 'Reference entities';
         case 'App catalog':
         case 'App catalog product':
-            return 'App catalogs';
+            return 'App catalogs [beta]';
```

Comment:
using parenthesis makes bootstrap affix plugin bug (the menu won't stick on screen)

Reply:
so maybe we should use [beta] instead of (beta) in other menus as well?

Reply:
as you wish

Reply:
done